app/controllers/unattended_controller.rb

```diff
@@ -211,18 +211,19 @@ def ip_from_request_env
   end
 
   def safe_render(template)
-    @template_name = 'Unnamed'
+    @unsafe_template  = template
     if template.is_a?(String)
-      @unsafe_template  = template
+      @unsafe_template_content  = template
```

Comment:
`lib/foreman/renderer.rb` `#template_name` still has a reference to `@unsafe_template`